src/main/scala/edu/cmu/ml/rtw/pra/features/PathFinder.scala

```diff
@@ -201,9 +201,10 @@ class GraphChiPathFinder(
     // and the rest of the columns make up the vector.
     fileUtil.readLinesFromFile(embeddingsFile).map(line => {
```

Comment:
There's a slightly more idiomatic way to do this in scala, using `flatMap` instead of a `filter`:

```scala
fileUtil.readLinesFromFile(embeddingsFile).flatMap(line => {
  val fields = line.split("\t");
  if (graph.hasEdge(fields(0))) {
    val relationIndex = graph.getEdgeIndex(fields(0))
    val vector = fields.drop(1).map(_.toDouble)
    Seq((relationIndex, new Vector(vector)))
  } else {
    Seq()
  }
}.toMap
```
This saves a tiny bit of memory and processing time, also, as you're not constructing vectors unless the edge is in the graph.